debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -208,7 +209,12 @@ public BinlogReader(String name, MySqlTaskContext context, HaltingPredicate acce
             }
         }
         client.setKeepAlive(context.config().getBoolean(MySqlConnectorConfig.KEEP_ALIVE));
-        client.setKeepAliveInterval(context.config().getLong(MySqlConnectorConfig.KEEP_ALIVE_INTERVAL_MS));
+        final long keepAliveInterval = context.config().getLong(MySqlConnectorConfig.KEEP_ALIVE_INTERVAL_MS);
+        client.setKeepAliveInterval(keepAliveInterval);
+        //Considering heartbeatInterval should be less than keepAliveInterval, we use the heartbeatIntervalFactor
+        //multiply by keepAliveInterval and set the result value to heartbeatInterval.The default value of heartbeatIntervalFactor
+        //is 0.8, and we believe the left time (0.2 * keepAliveInterval) is enough to process the packet received from the MySQL server.
+        client.setHeartbeatInterval((long)(keepAliveInterval*heartbeatIntervalFactor));
```

Comment:
```suggestion
        client.setHeartbeatInterval((long) (keepAliveInterval * heartbeatIntervalFactor));
```

Reply:
@gunnarmorling Thanks  a lot  for your remind , and I will change the code to fit the CheckStyle validatition and rework the commit.